pkg/pillar/types/diskmetrics.go

```diff
@@ -20,16 +20,27 @@ var ReportDiskPaths = []string{
 
 // ReportDirPaths  Report directory usage for these paths
 var ReportDirPaths = []string{
-	PersistDir + "/downloads", // XXX old to be removed
-	PersistDir + "/img",       // XXX old to be removed
+	PersistConfigDir,
+	PersistStatusDir,
+	CertificateDirname,
+	SealedDirName,
+	ClearDirName,
+	PersistDebugDir,
+	PersistInstallerDir,
+	IngestedDirname,
+	SnapshotsDirname,
+	PersistCachePatchEnvelopes,
+	PersistCachePatchEnvelopesUsage,
 	PersistDir + "/containerd",
 	PersistDir + "/tmp",
 	PersistDir + "/log",
 	PersistDir + "/newlog",
```

Comment:
This one is also defined as a constant...
https://github.com/lf-edge/eve/blob/68b0c3905c13f0296fd5f621e2fcc1fb88354efd/pkg/pillar/types/locationconsts.go#L92
Maybe I would keep it as it is now, to not deploy the necessary fix.

But it would be nice to review the code for the places where we use different dirs and ensure they are synced with the constants - the centralized approach for dirs definition could simplify code understanding.

Reply:
Oops I missed that one in the grep output. Will fix.

But I'm also thinking of doing a directory walk to report other large files and directories. TBD.

Reply:
Just in case, I've created a GH Issue #3682. It's a good task for someone new to the codebase to take it, so I've labelled it with the "good first issue" tag.